### PR TITLE
docker: Add docker buildx functionality to dockernode playbook for arm32 containers

### PR DESCRIPTION
**Pull request by Haroon-Khel**
- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [x] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

Ref https://github.com/adoptium/infrastructure/issues/3579

Adds ability to build arm32 containers using docker buildx. Updates docs accordingly

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ The `dockerhost.yml` playbook can deploy single, multiple and duplicate containe
 
 will deploy 1 Ubuntu 22.04 container, 1 Alpine 3.19 container and 3 Ubuntu 18.04 containers.
 
+If you would like to build an arm32 container on an arm64 dockerhost, pass the `build_arm` variable:
+
+```
+ansible-playbook -u root -i <host-file> AdoptOpenJDK_Unix_Playbook
+/dockernode.yml -t "deploy" -e "docker_images=u2204 build_arm=yes"
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/README.md:40]: Would it be better to say arm32?

Comment (sxa) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/README.md:40]: Yep I agree - arm is too ambiguous to use here IMHO

Comment (Haroon-Khel) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/README.md]: Done

**Review by sxa — COMMENTED** [review]
Assuming this has been tested with the latest changes LGTM to me.

**Comment by Haroon-Khel** [comment]
> Assuming this has been tested with the latest changes LGTM to me.

Yes the latest changes have been tested 👍🏻

**Review by steelhead31 — APPROVED** [review]
Looks good.

**Review by sxa — APPROVED** [review]
Apologies - I thought my last comment included an approval.

**Pull request merged**